src/components/header/search.js

```diff
@@ -34,7 +36,7 @@ export default class Search extends Component {
 				docsearchInstance = docsearch({
 					apiKey: config.docsearch.apiKey,
 					indexName: config.docsearch.indexName,
-					inputSelector: `#${this.id}`
+					inputSelector: this.input.current
```

Comment:
Note to self: check if passing a node instead of a selector works with docsearch

Reply:
Just checked and it works :tada:

Reply:
lol yeah their docs don't specify but I dove through the source to make sure this would work 😂